src/tab.js

```diff
@@ -187,3 +187,45 @@ let updateAction = (tabId, delay = null) => {
     });
   }, delay));
 };
+
+
+util.addListener(browser.commands.onCommand, async (command) => {
+  try {
+    switch (command) {
+      case 'toggle-tab-status':
+        await toggleTabStatus();
+        break;
+      case 'toggle-global-status':
+        await toggleGlobalStatus();
+        break;
+      case 'open-options':
+        await openOptionsPage();
+        break;
+    }
+  } catch (e) {
+    console.error('Command error:', command, e);
+  }
+});
+
+async function toggleTabStatus() {
+  const tabs = await browser.tabs.query({ active: true, currentWindow: true });
+  if (!tabs?.[0]) return;
```

Comment:
To ensure compatibility with older browsers, simply use:

```suggestion
  if (!tabs[0]) return;
```